packages/components/pie-button/src/button.scss

```diff
@@ -1,160 +1,126 @@
     }
 
-    @if $outline-color != null {
-        border: 1px solid $outline-color;
+    & .o-btn:active:not(:disabled) {
+        background-color: hsl(var(#{$bg-color}-h), var(#{$bg-color}-s), calc(var(#{$bg-color}-l) - var(--dt-color-active-01)));
     }
 }
 
 // Base button styles
 .o-btn {
-    border-radius: $button-border-radius;
+    min-block-size: var(--btn-min-block-size);
+    padding: var(--btn-padding);
     border: none;
-    font-family: $button-font;
-
+    border-radius: var(--btn-border-radius);
+    outline: none;
+    background-color: var(--btn-bg-color);
+    font-family: var(--btn-font-family);
+    font-size: var(--btn-font-size);
+    font-weight: var(--btn-font-weight);
+    color: var(--btn-text-color);
+    line-height: var(--btn-line-height);
     cursor: pointer;
     user-select: none;
-    outline: none;
 
     &:focus-visible {
-        outline: 2px solid $button-focus;
-    }
-}
-
-// Sizes
-$button-sizes: (
-    'xsmall': (
-        'min-block-size': $button-height-xs,
-        'padding-block': 6px,
-        'padding-inline': dt.$spacing-b,
-        'font-size': dt.$font-interactive-xs-size,
-        'font-weight': dt.$font-interactive-xs-weight,
-        'line-height': dt.$font-interactive-xs-line-height
-    ),
-    'small-productive': (
-        'min-block-size': $button-height-s,
-        'padding-block': dt.$spacing-b,
-        'padding-inline': dt.$spacing-d,
-        'font-size': dt.$font-interactive-s-size,
-        'font-weight': dt.$font-interactive-s-weight,
-        'line-height': dt.$font-interactive-s-line-height
-    ),
-    'small-expressive': (
-        'min-block-size': $button-height-s,
-        'padding-block': 6px,
-        'padding-inline': dt.$spacing-d,
-        'font-size': dt.$font-interactive-l-size,
-        'font-weight': dt.$font-interactive-l-weight,
-        'line-height': dt.$font-interactive-l-line-height
-    ),
-    'medium': (
-        'min-block-size': $button-height-m,
-        'padding-block': 10px,
-        'padding-inline': dt.$spacing-e,
-        'font-size': dt.$font-interactive-l-size,
-        'font-weight': dt.$font-interactive-l-weight,
-        'line-height': dt.$font-interactive-l-line-height
-    ),
-    'large': (
-        'min-block-size': $button-height-l,
-        'padding-block': 14px,
-        'padding-inline': dt.$spacing-e,
-        'font-size': dt.$font-interactive-l-size,
-        'font-weight': dt.$font-interactive-l-weight,
-        'line-height': dt.$font-interactive-l-line-height
-    )
-);
-
-@each $size, $values in $button-sizes {
-    .o-btn--#{$size} {
-        @include button-sizing(
-            map-get($values, 'min-block-size'),
-            map-get($values, 'padding-block'),
-            map-get($values, 'padding-inline'),
-            map-get($values, 'font-size'),
-            map-get($values, 'font-weight'),
-            map-get($values, 'line-height')
-        );
+        outline: 2px solid var(--button-focus);
```

Comment:
might be worth creating a ticket to apply the new focus outline styles to this

Reply:
A comment too if possible.